### ISSUE TITLE
Delete user function should delete all notes

### ISSUE THREAD

**Issue by karolis-arbaciauskas** [AWSinAction/apigateway#5]
When you are deleting user shouldn't you delete all notes chained with user?

**Comment by michaelwittig** [comment]
Yes, that would be better. Since this is an example it does not cover that functionality. But feel free to add it.
